modules/microshift-storage-backup-volume-snapshots.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// microshift/volume-snapshots-microshift.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-storage-backup-vol-snaps_{context}"]
+= Backing up a volume snapshot
+
+Snapshots of data from applications running on a {product-title} cluster are created as read-only logical volumes (LVs) located on the same devices as the original data. All local volumes must be manually mounted before they can be copied as persistent volumes (PVs) and used as backup copies. To use a snapshot of a {product-title} storage volume as a backup, you can locate it on the local host and then move it to a secure location. To find specific snapshots and copy them, use the following procedure.
+
+.Prerequisites
+
+. You have root access to the host machine.
+. You have an existing volume snapshot.
+
+.Procedure
+
+. Get the `volumeSnapshot.status.volumeSnapshotContentName` by using the following command syntax:
```

Comment:
+1. It was initially unclear to me that you're using the value from the first step to populate the value in the second step command. Especially because the first command calls it "volumeSnapshotContentName" and the second command calls it "<retrieved volume identity>".

I'd recommend to name the replaceable value the same as the field name, and then also mention in the step text that you use it.